pkgs/development/python-modules/xlsx2html/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, python3Packages, fetchPypi }:
```

```suggestion
{ lib, pythonPackages, fetchPypi }:
```
This breaks the package on python 3.7, 3.9 and 3.10. ``disabled = isPy27;``